_posts/elements/2015-05-12-forms.md

```diff
@@ -53,22 +53,22 @@ info: Forms allow users to interact with the interface, set preferences, and ent
                 </div>
             </div>
             <div class="field">
-                <label class="field-label field-required" for="email_address_default_suc_helper">Email address (default, has success)</label>
-                <input class="field-input input-text has-success" type="email" id="email_address_default_suc_helper" name="email_address_default_suc_helper" placeholder="e.g. jane@email.com" value="jane@domain.com" aria-describedby="email_address_default_suc_helper" required>
-                <div class="field-hint" id="email_address_default_suc_helper">
+                <label class="field-label field-required" for="email-address-default-suc-helper">Email address (default, has success)</label>
+                <input class="field-input input-text has-success" type="email" id="email-address-default-suc-helper" name="email-address-default-suc-hint" placeholder="e.g. jane@email.com" value="jane@domain.com" aria-describedby="email-address-default-suc-hint" required>
```

Comment:
Please do not abbreviate ``suc``? Let's match the ``success`` label/term used elsewhere in the DOM.